eva-accession-import-automation/data_ops.py

```diff
@@ -39,3 +39,48 @@ def get_species_pg_conn_info(pg_metadata_dbname, pg_metadata_user, pg_metadata_h
     pg_cursor.close()
     pg_conn.close()
     return species_set
+
+
+def get_species_info(pg_metadata_dbname, pg_metadata_user, pg_metadata_host,
+                     assembly_accession):
+    """
+    Get information about an assembly accession
+
+    :param pg_metadata_dbname: Metadata database name
+    :param pg_metadata_user: Metadata user name
+    :param pg_metadata_host: Host where the metadata database resides
+    :param assembly_accession: species to query, e.g. GCA_000001735.1
+    :return: Dictionary with species information
+    """
+    pg_conn = psycopg2.connect("dbname='{0}' user='{1}' host='{2}'".
+                               format(pg_metadata_dbname, pg_metadata_user, pg_metadata_host))
+    pg_cursor = pg_conn.cursor()
+    pg_cursor.execute("select database_name, tax_id from dbsnp_ensembl_species.import_progress i "
```

Comment:
Have you made sure that all the tax_ids in import_progress are consistent and filled out?

Reply:
all tax_ids in import progress match the database_name in import_progress

Reply:
also, I removed this function by adding the --species parameter